spec/unit/dao/entities_schemas_spec.lua

```diff
@@ -84,15 +84,50 @@ describe("Entities Schemas", function()
       assert.equal("At least a 'request_host' or a 'request_path' must be specified", errors.request_host)
     end)
 
-    it("should complain if `request_host` is an empty string", function()
+    it("should complain if request_host is an empty string", function()
       local t = {request_host = "", upstream_url = "http://mockbin.com"}
 
       local valid, errors = validate_entity(t, api_schema)
       assert.False(valid)
-      assert.equal("request_host has an invalid value", errors.request_host)
+      assert.equal("At least a 'request_host' or a 'request_path' must be specified", errors.request_host)
+      assert.equal("At least a 'request_host' or a 'request_path' must be specified", errors.request_path)
+    end)
+
+    it("should complain if request_path is an empty string", function()
+      local t = {request_path = "", upstream_url = "http://mockbin.com"}
+
+      local valid, errors = validate_entity(t, api_schema)
+      assert.False(valid)
+      assert.equal("At least a 'request_host' or a 'request_path' must be specified", errors.request_host)
       assert.equal("At least a 'request_host' or a 'request_path' must be specified", errors.request_path)
     end)
 
+    it("should not accept an invalid request_host", function()
+      local invalids = {"/mockbin", ".mockbin", "mockbin.", "mockbin.f", "mock;bin",
+                        "mockbin.com-org", "mockbin.com/org", "mockbin.com_org"}
+
+      for _, v in ipairs(invalids) do
+        local t = {request_host = v, upstream_url = "http://mockbin.com"}
+        local valid, errors = validate_entity(t, api_schema)
+        assert.equal("Invalid value: "..v, (errors and errors.request_host or ""))
+        assert.falsy(errors.request_path)
+        assert.False(valid)
+      end
+    end)
+
+    it("should accept valid request_host", function()
+      local valids = {"hello.com", "hello.fr", "test.hello.com", "1991.io", "hello.COM",
+                      "HELLO.com", "123helloWORLD.com", "mockbin.123", "mockbin-api.com",
+                      "hello.abcd", "mockbin_api.com"}
```

Comment:
@Tieske @thefosk Could you please review the valid and invalid cases pointed out by those tests?